specific_use_cases/DFU_work_in_progress/station_8760s.ipynb

```diff
@@ -0,0 +1,627 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d356ce4-e07e-4a0a-933a-f2401e64c420",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "median_year.drop(['scenario','simulation']).hvplot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d22d6e66-7216-4f57-8a18-6ccd9f9cc9aa",
+   "metadata": {},
+   "source": [
+    "#### To understand what this is, compare with other 8760s.\n",
+    "\n",
+    "We will compare this 8760 (closest to the median) with the statistical median itself, and the range of other 8760s, looking one station at a time.\n",
+    "\n",
+    "First, a little work gathering and reshaping the data to plot:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e79dde7-21fc-49d8-bb12-ac50c680dc1e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "def median_by_hour(y):\n",
+    "    by_hour = y.groupby('time.hour').median('time')\n",
+    "    generic_year_day = pd.to_datetime('1981'+str(y['time.month'].values[0]).zfill(2)+\n",
+    "                                      str(y['time.day'].values[0]).zfill(2))\n",
+    "    by_hour['time'] = pd.date_range(generic_year_day,periods=24,freq='H')\n",
+    "    by_hour = by_hour.drop('hour')\n",
+    "    return by_hour\n",
+    "\n",
+    "stat_median = data.groupby('time.dayofyear').apply(median_by_hour)\n",
+    "stat_median['time'] = np.arange(1,8761)\n",
+    "stat_median = stat_median.to_array('station')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6981cfee-64ce-4329-8084-45661f257d67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "median_year_by_station = median_year.to_array('station')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1db850a8-64f3-402c-a72e-6876afc8373c",
+   "metadata": {},
+   "source": [
+    "Some rearrangement of the initial data, by year:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d6bd773-3f16-4a0e-8048-5e0f19d7cf4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all8760s = data.to_array('station')\n",
+    "\n",
+    "def number_years(y):\n",
+    "    y['year'] = y['time.year'].values[0]\n",
+    "    y['time'] = np.arange(1,8761)\n",
+    "    return y\n",
+    "\n",
+    "all8760s = all8760s.groupby('time.year').apply(number_years)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f24d4452-8952-44cc-b5be-e3814c6bf261",
+   "metadata": {},
+   "source": [
+    "Now finally plot it all together. (Note that this plot is interactive and you can zoom in and out.) Initial plotting (and each time you switch station) may take a moment.\n",
+    "\n",
+    "Samples from all 8760's are plotted in light grey, the statistical medians for each of the 8760 hours of the year are in black, and the single 8760 with the smallest euclidean distance from that statistical median is shown on top in the thin line."
```

Comment:
This plot hangs a lot for me when I try to switch stations and my browser tab goes unresponsive (to be fair, I did pick 1980-2050 originally so I have a lot of years in my data object). Maybe we should flag that for the user in the text

Reply:
Yes, I've added a warning to stick with the historical period, both for speed and because this notebook is not accounting for trend in the data.